test/karma.conf.js

```diff
@@ -1,196 +1,41 @@
-module.exports = function(config) {
-  // build out a name for browserstack
-  // {TRAVIS_BUILD_NUMBER} [{TRAVIS_PULL_REQUEST} {PR_BRANCH}] {TRAVIS_BRANCH}
-  var browserstackName = process.env.TRAVIS_BUILD_NUMBER;
-
-  if (process.env.TRAVIS_PULL_REQUEST !== 'false') {
-    browserstackName += ' ' + process.env.TRAVIS_PULL_REQUEST + ' ' + process.env.TRAVIS_PULL_REQUEST_BRANCH;
-  }
-
-  browserstackName +=  ' ' + process.env.TRAVIS_BRANCH;
-
-  // Creating settings object first so we can modify based on travis
-  var settings = {
-    basePath: '',
-
-    frameworks: ['browserify', 'qunit', 'detectBrowsers'],
-    autoWatch: false,
-    singleRun: true,
-
-    // Compling tests here
-    files: [
-      '../dist/video-js.css',
-      '../test/globals-shim.js',
-      '../test/unit/**/*.js',
-      '../build/temp/browserify.js',
-      '../build/temp/webpack.js',
-      { pattern: '../src/**/*.js', watched: true, included: false, served: false }
-    ],
-
-    preprocessors: {
-      '../test/**/*.js': [ 'browserify' ]
-    },
-
-    browserify: {
-      debug: true,
-      plugin: ['proxyquireify/plugin'],
-      transform: [['babelify', {
-        "presets": [
-          [
-            "@babel/preset-env",
-            {
-              "loose": true
-            }
-          ]
-        ]
-      }]]
-    },
-
-    plugins: [
-      'karma-qunit',
-      'karma-chrome-launcher',
-      'karma-firefox-launcher',
-      'karma-ie-launcher',
-      'karma-opera-launcher',
-      'karma-safari-launcher',
-      'karma-safaritechpreview-launcher',
-      'karma-browserstack-launcher',
-      'karma-browserify',
-      'karma-coverage',
-      'karma-detect-browsers',
-    ],
-
-    detectBrowsers: {
-      enabled: false,
-      usePhantomJS: false
-    },
-
-    reporters: ['dots'],
-
-    // web server port
-    port: 9876,
-
-    // cli runner port
-    runnerPort: 9100,
-    colors: true,
-    logLevel: config.LOG_INFO,
-    captureTimeout: 300000,
-    browserNoActivityTimeout: 300000,
-    browserDisconnectTimeout: 300000,
-    browserDisconnectTolerance: 3,
-
-    browserStack: {
```

Comment:
the project, name, and build, don't all seem to be configurable or that they'll be set to the correct value.

Reply:
Three are different here things here:
1. The project name will be `video.js` instead of `Video.js`, i'm not sure if that is an issue
1. Missed a value above in  videojs-generate-karma. https://github.com/videojs/videojs-generate-karma-config/pull/9
1. The browserStack project name will be lowercase, and the `build` value will be `process.env.TRAVIS_BUILD_NUMBER || process.env.BUILD_NUMBER`, but I think that makes more sense than the previous value

Reply:
Renaming it isn't huge but it will mean that previous runs will be considered separate from the new runs. It would be nice if we can keep the name.
Also, I set the name as the build specifically so that there will be more info there https://github.com/videojs/video.js/pull/4903#issuecomment-360634649. The Browserstack UI will show the build value and not the name, so, having the info from the full name, is useful in narrowing down the specific build you want to look at.

Also, since we no longer care about IE8, we should turn off video recording of the browserstack session, it'll speed things up a lot.

Reply:
https://github.com/videojs/videojs-generate-karma-config/pull/12

Reply:
Looks like project names are not case sensitive, since this build shows up with our old builds.